app/views/admin/action_pages/_petition_fields.html.erb

```diff
@@ -8,4 +8,17 @@
     <%= sf.label :goal %>
     <%= sf.number_field :goal %>
   </div>
+
+  <div class="form-item">
+    <%= sf.label :enable_affiliations do %>
+      Add petition targets <span class="hint">(ex. universities or cities)</span>
+    <% end %>
+    <%= sf.check_box :enable_affiliations %>
+  </div>
 <% end %>
+
+<div id="affiliations-enabled" <%= 'class="hidden"' unless @actionPage.petition.enable_affiliations %>>
+  <%= link_to "Edit petition targets and affiliations",
+    edit_targets_admin_action_page_institutions_path(@actionPage),
+    target: "_blank" %>
```

Comment:
this link will only work on persisted action pages, should probably surface this link better (maybe in the sidebar on edit, only when affiliations are enabled?)